File: Day_4/Java/Palindrome.java
public class Palindrome{


	public static boolean[] checkPalindrome(String[] words){
		boolean[] newArray = new boolean[words.length];
		for(int count = 0; count < words.length; count++){
		String palindrome = "";

			for(int counter = 0; counter < words[count].length(); counter++){
				palindrome = words[count].charAt(counter) + palindrome;
				}

			if (words[count].equals (palindrome))
				newArray[count] = true;
			else
				newArray[count] = false;
		}
		System.out.print(newArray);
		return newArray;
		
		}
}

File: Day_4/Python/palindrome.py
def is_palindrome(words):
	new_element= words
	count = 0
	for elements in words:
		palindrome = ""
		for element in elements:
			palindrome = element + palindrome

		if elements == palindrome:
			new_element[count] = True;
		else:
			new_element[count] = False;
		count+=1
	return new_element
	
words  = ["madam", "kali", "hello", "noon"]
print(is_palindrome(words))